src/events/messageCreate.ts

```diff
@@ -74,14 +82,39 @@ const punishSpammersAndTrolls = async (
 };
 
 /**
- * Convert any pdfs sent in the #resumes channel to an image.
+ * Convert any pdfs sent in the #resumes channel to an image,
+ * nuke message and DM user if no attachment is found or attachment is not PDF
  */
 const convertResumePdfsIntoImages = async (
+  client: Client,
   message: Message,
 ): Promise<Message<boolean> | undefined> => {
   const attachment = message.attachments.first();
-  // If no resume pdf is provided, do nothing
-  if (!attachment || attachment.contentType !== 'application/pdf') return;
+
+  // If no resume pdf is provided, nuke message and DM user about why their message got nuked
+  if (!attachment || attachment.contentType !== 'application/pdf') {
```

Comment:
We need to support images as well, since many people upload images directly instead of pdfs. 
Create another condition to handle images and create threads with the same image.

Reply:
I added support for images. It seems to work for most image extension, save for two that I'm not very confident of. If you view my testing channel #di-test-resume, you can see that AVIF images don't get a preview like other extensions. And I couldn't find a sample HEIF image to test for iPhone users.

Reply:
Actually, I've found sample HEIF images. Same issue with AVIF, no previews.

Reply:
Could you try and convert the HEIF and AVIF images into JPG/PNG and then send them into the thread?
Also even if there is an image preview in the message itself, send that in the thread for consistency.

Reply:
So I managed to convert HEIF/HEIC images to JPG. This was done by using a package that runs shell command to do the conversion. For an HEIF/HEIC image of 2.5 MB, the resulting JPG is 10 MB, which is fine for uploading to Discord. This appears to be common among the converters I tested, and this one is the most efficient one. Then again, I'm willing to bet that resumes won't break this limit.

I tried some other packages that doesn't require CLI but they all don't work with TS, so I got desperate and opted to the CLI ones, which actually worked.

Reply:
That's fine. Its going to be rare anyway for people to upload heif/heic.